packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -0,0 +1,88 @@
+import { LitElement, customElement, html, property } from 'lit-element';
+// import { Base } from '@sfx/base';
+import '@sfx/autocomplete';
+import { SAYT_EVENT } from './events';
+
+/**
+ * The `sfx-sayt` component is responsible for displaying and hiding the
+ * `sfx-autocomplete` and `sfx-products` components.
+ */
+@customElement('sfx-sayt')
+// TODO: look into the Base class so this class can extend it again. The
+// issue that we're seeing is related to changing properties and having
+// the DOM update accurately in response to those property changes.
+export default class Sayt extends LitElement {
+  /**
+   * Determines if the `sfx-autocomplete` component will be hidden or not.
+   */
+  @property({ type: Boolean, reflect: true }) hideAutocomplete = false;
+  /**
+   * Determines the visibility of the `sayt` component.
+   */
+  @property({ type: Boolean, reflect: true }) show = false;
```

Comment:
Nitpick, but is "visible" a better word to use given its precedent in CSS (`visibility`) and jQuery (`:visible`)? It's also an adjective rather than a verb, which seems to be the convention for [boolean attributes](https://gist.github.com/ArjanSchouten/0b8574a6ad7f5065a5e7).